apps/experiments/management/commands/migrate_nonpipeline_to_pipeline_experiments.py

```diff
@@ -0,0 +1,231 @@
+        failed_count = 0
+
+        for experiment in experiments_to_convert:
+            try:
+                with transaction.atomic():
+                    self._convert_experiment(experiment)
+                    converted_count += 1
+                    self.stdout.write(self.style.SUCCESS(f"Success: {experiment.name}"))
+            except Exception as e:
+                failed_count += 1
+                self.stdout.write(self.style.ERROR(f"FAILED {experiment.name}: {str(e)}"))
+
+        self.stdout.write(
+            self.style.SUCCESS(f"\nMigration is complete!: {converted_count} succeeded, {failed_count} failed")
+        )
+
+    def _get_experiment_type(self, experiment):
+        if experiment.assistant:
+            return "Assistant"
+        elif experiment.llm_provider:
+            return "LLM"
+        else:
+            return "Unknown"
+
+    def _convert_experiment(self, experiment):
+        experiment_type = self._get_experiment_type(experiment)
+
+        if experiment_type == "Assistant":
+            pipeline = self._create_assistant_pipeline(experiment)
+        elif experiment_type == "LLM":
+            pipeline = self._create_llm_pipeline(experiment)
+        else:
+            raise ValueError(f"Unknown experiment type for experiment {experiment.id}: {experiment_type}")
+
+        experiment.pipeline = pipeline
+
+        # Clear the old type-specific fields
+        if experiment_type == "Assistant":
+            experiment.assistant = None
+        elif experiment_type == "LLM":
+            experiment.llm_provider = None
+            experiment.llm_provider_model = None
+
+        experiment.save()
+
+    def _get_chatbots_flag_team_ids(self):
+        chatbots_flag = Flag.objects.get(name="flag_chatbots")
+        return list(chatbots_flag.teams.values_list("id", flat=True))
+
+    def _create_pipeline_with_node(self, experiment, node_type, node_label, node_params):
+        """Create a pipeline with start -> custom_node -> end structure."""
+        pipeline_name = f"{experiment.name} Pipeline"
+        pipeline = Pipeline.objects.create(team=experiment.team, name=pipeline_name, data={"nodes": [], "edges": []})
+        start_id = str(uuid4())
+        middle_id = str(uuid4())
+        end_id = str(uuid4())
+
+        start_node = FlowNode(
+            id=start_id,
+            type="startNode",
+            position={"x": 100, "y": 200},
+            data=FlowNodeData(id=start_id, type=StartNode.__name__, params={"name": "start"}),
+        )
+
+        middle_node = FlowNode(
+            id=middle_id,
+            type="pipelineNode",
+            position={"x": 400, "y": 200},
+            data=FlowNodeData(
+                id=middle_id,
+                type=node_type,
+                label=node_label,
+                params=node_params,
+            ),
+        )
+
+        end_node = FlowNode(
+            id=end_id,
+            type="endNode",
+            position={"x": 700, "y": 200},
```

Comment:
This is what it looks like when I create a pipeline with the command. Maybe the x-axis can do with 50 or a 100 more?

![Screenshot from 2025-06-06 08-31-54](https://github.com/user-attachments/assets/ac11fa25-a202-40ba-93fd-72c58fe7b1ee)